CMakeLists.txt

```diff
@@ -257,12 +264,14 @@ if (CGNS_ENABLE_HDF5)
   set (HDF5_PACKAGE_NAME ${SEARCH_PACKAGE_NAME})
 
   if (HDF5_FOUND)
+    if (WIN32)
```

Comment:
```suggestion
    if (NOT DEFINED HDF5_INCLUDE_DIRS)
```

Reply:
Yes, that is probably a better option.  It looks like some `FindHDF5.cmake` packages are setting `HDF5_INCLUDE_DIR` instead of `HDF5_INCLUDE_DIRS`.  Based on the CMake documentation, `HDF5_INCLUDE_DIR` was deprecated as of CMake-2.8.1, but it looks like the build is still using the older versions in some cases.